.cascade-code/left-nav-refactor/_components/left_nav/left_nav_index_format

```diff
@@ -0,0 +1,164 @@
+## Format used by index pages with the brute-force block that loads the entire file tree.
```

Comment:
So this would be the new default? And even if it doesn't fix the slow-load issue, you said that you still managed to cut the code in half, yes?

Reply:
Yes this is the new default. The names need to be altered a bit. I want to name this one left_nav_format, so I have to rename the one that's currently called that. And that one is the format to be used with the smaller block on non-index pages. It's basically siblings only, so maybe something with that? Maybe we change the other two to `left_nav_children_only` and `left_nav_siblings_only`?

Reply:
The current left nav format in production is 220 lines and the refactored one is about 130 right now